demo/java-demo-client/src/main/java/org/cloudfoundry/abacus/demo/model/Space.java

```diff
@@ -0,0 +1,66 @@
+package org.cloudfoundry.abacus.demo.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.StringJoiner;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Space {
+
+  @JsonProperty("space_id")
+  private String spaceID;
+
+  private Resource resources[];
```

Comment:
Add `@JsonProperty` for consistency and refactor protection.

Reply:
Done